pm_icecon/bt/cli.py

```diff
@@ -12,8 +15,419 @@
 from pm_icecon.util import standard_output_filename
 
 
+def get_filename_attrs_a2l1c(fn):
+    # Extract information from the filename
+    # Assumes the base filename is in the form:
+    #  bt_NH_20220101_a2l1c_6.25km.nc
+    base_filename = fn.name.replace('.nc', '')
+    # alg, hem, ymd, tframe, tbsrc, res = base_filename.split('_')
+    alg, hem, ymd, _, tbsrc, res = base_filename.split('_')
```

Comment:
Note that I don't think we need to address this for this PR, just something to be aware of for the future. It's difficult to trust code like this, in part because a change to the filename (e.g., adding another component separated by the `_`) would result in an ambiguous error about unpacking arguments. 

Ideally key attributes of the data will be encoded along with the data from `pm_tb_data`.